app/uk/gov/hmrc/submissiontracker/controllers/action/AccountAccessControl.scala

```diff
@@ -21,122 +21,64 @@ import play.api.libs.json.Json
 import play.api.mvc.{ActionBuilder, Request, Result, Results}
 import uk.gov.hmrc.api.controllers._
 import uk.gov.hmrc.domain.Nino
-import uk.gov.hmrc.http._
 import uk.gov.hmrc.play.HeaderCarrierConverter
-import uk.gov.hmrc.play.auth.microservice.connectors.ConfidenceLevel
-import uk.gov.hmrc.submissiontracker.connector._
-import uk.gov.hmrc.submissiontracker.controllers.{ErrorNinoInvalid, ErrorUnauthorizedNoNino, ForbiddenAccess}
+import uk.gov.hmrc.submissiontracker.controllers.{ErrorUnauthorizedNoNino, _}
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.Future
 
 case object ErrorUnauthorizedMicroService extends ErrorResponse(401, "UNAUTHORIZED", "Unauthorized to access resource")
 case object ErrorUnauthorizedWeakCredStrength extends ErrorResponse(401, "WEAK_CRED_STRENGTH", "Credential Strength on account does not allow access")
 
 
-trait AccountAccessControl extends Results {
+trait AccountAccessControl extends Results with Authorisation {
```

Comment:
I think this is a better approach than the one I've used elsewhere of mixing the AccessControl and Auth concerns into 1 object. Probably should refactor towards this pattern as part of the guide upgrade whenever we get round to playing this.